### DISCUSSION TITLE
contact form

### DISCUSSION THREAD

**Discussion by kendrickcc** [Lednerb/bilberry-hugo-theme#478] [Q&A]
I'm new to Hugo. 

I'm trying to add a formspree.io for to a contact page, that will appear in the top banner. When I add the shortcode and call it in a page, the form does not appear. 

But what I think might be beneficial is to add a parameter in config.toml for the formspree form id, then the shortcode and page would all tie together. 

Thoughts? 
Be possible to provide the code for that?

Thanks in advance.

**Comment by igor-baiborodine** [comment]
@kendrickcc, could you please provide me with a link to the GitHub repo? I will investigate this issue in my local dev.

**Comment by kendrickcc** [comment]
Repo available at https://github.com/kendrickcc/ckp-test-site/.
The contact-form shortcode contains my formspree id but I'd like to see this moved to the config.toml file as a parameter similar to discuss/giscus etc..

**Comment by igor-baiborodine** [comment]
@kendrickcc, moving to "Discussions" since it's not related to implemented by the theme issues but to the integration with a third-party provider.

**Comment by igor-baiborodine** [comment]
@kendrickcc, it looks like your contact form works fine in my local dev. Please see my [pull request](https://github.com/kendrickcc/ckp-test-site/pull/1) for moving the Formspree ID to the `config.toml` file.

**Reply by kendrickcc** [reply]
Much appreciated.
